conf/airframes/ardrone2_optitrack.xml

```diff
@@ -144,11 +129,51 @@
     <define name="DDGAIN_R" value="300"/>
   </section>
 
+  <section name="STABILIZATION_ATTITUDE_INDI" prefix="STABILIZATION_INDI_">
+    <!-- setpoints -->
+    <define name="SP_MAX_PHI" value="60." unit="deg"/>
```

Comment:
Apparently, they are defined with the correct prefix somewhere later in this file. So, these are ambiguous and I'll remove them here.